### ISSUE TITLE
Better integration into Redmine’s native time tracking

### ISSUE THREAD

**Issue by rbugajewski** [hicknhack-software/redmine_time_tracker#164]
It would be really great if the plugin’s UI (e.g. Reports) would also display time logs from Redmine’s native time tracking functionality. This currently works in one direction (the plugin’s booked entries are for example shown in Redmine’s report), but not the other way around. From my limited knowledge of the Redmine source code I guess that there’s simply no start / stop date / time model and that it only saves the day + amount of time spent. The plugin added this in separate tables. Please correct me if I’m wrong. One could solve this feature request by marking the time bookings with a dash or something similar for that purpose.

I love to work with the TimeTracker plugin, but I also use mobile apps or other plugins like Redmine Harvest integration that work with Redmine’s native functionality. This way you still could use 3rd party tools and see the project, activity, date, issue, comments & time spent in the Overview, Bookings & Reports sections.

Is this something doable or would it be – code-wise – very messy?

**Comment by jume-dev** [comment]
Alright, for everyone who still wants to use our plugin with the enhanced mechanism, we finally managed to clean things up and created a completely [new version](https://github.com/hicknhack-software/redmine_hourglass). While we don't have all features here requested in it yet, we have atleast the manually booking in place, but the code is now way better than before and adding new features is faster done.
The use of redmines own time entries without attached booking is planned and will be eventually added some day, but we also plan on creating an own desktop and mobile app to make use of the new api we designed.
